From ed1deee11b4494c3b4cd80e806f83ebd697591a9 Mon Sep 17 00:00:00 2001
From: Andreas Sturmlechner <asturm@gentoo.org>
Date: Mon, 20 Oct 2025 12:27:58 +0200
Subject: [PATCH] Raise minimum CMake version to 3.16

CMake 3.31 warns about compat. for <3.10 being removed in the future.
3.16 aligns with most of ECM revdeps while some ECM modules already
warn about using <3.16 minimum.

Signed-off-by: Andreas Sturmlechner <asturm@gentoo.org>
---
 CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 350e7608..65efaa11 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,4 @@
-# cmake CXX_STANDARD needs min cmake 3.1
-# https://stackoverflow.com/questions/10984442/how-to-detect-c11-support-of-a-compiler-with-cmake/20165220#20165220
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.16)
 
 #set( KTECHLAB_DEVELOPER_BUILD "true" )
 
-- 
GitLab

